tests/suites/test_suite_ssl.function

```diff
@@ -12,7 +12,8 @@
 
 #define SSL_MESSAGE_QUEUE_INIT      { NULL, 0, 0, 0 }
 
-#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C) && \
+#if (!defined(MBEDTLS_SSL_PROTO_TLS1_2)) && \
```

Comment:
A few things:
- Are we tracking the issue with `vsnprintf()` somewhere? (Or will it just go away when we officially stop supporting visual studio 2013?)
- Do we have a plan to ensure we don't forget to remove this hack when it's no longer needed?
- Thanks for explaining in the commit message, but should we add a comment too? Right now if someone read this and wonders why, they need to use git blame to find the commit message.

Reply:
> A few things:
> 
>     * Are we tracking the issue with `vsnprintf()` somewhere? (Or will it just go away when we officially stop supporting visual studio 2013?)

I have a PR demonstrating the issue: #8725. When we move to Windows 2017 instead of Windows 2013 we will how it goes there.

> 
>     * Do we have a plan to ensure we don't forget to remove this hack when it's no longer needed?

Created #8772 to not forget about it at the end of the quarter. Hopefully this will be resolved by then. Otherwise I will probably have to create a specific issue for that. 

>     * Thanks for explaining in the commit message, but should we add a comment too? Right now if someone read this and wonders why, they need to use git blame to find the commit message.

Added a comment.

Reply:
Ok. See also https://github.com/Mbed-TLS/mbedtls/issues/8770 that I created (and added to the main 3.6 EPIC) in the meantime.